### ISSUE TITLE
memcached-php-80 missing

### ISSUE THREAD

**Issue by robchett** [brefphp/extra-php-extensions#268]
Looks like you've recently added a memcached build for 8.1, could it be availabled for us people stuck on 8.0?

**Comment by Nyholm** [comment]
Of course. Send a PR to modify this file. https://github.com/brefphp/extra-php-extensions/blob/master/layers/memcached/config.json


Then hope the CI is green :)
